Testscripts/Linux/L3Cache-Verify.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# Description:
+#	This script was created to automate the testing of L3 Cache spread.
+# The L3 cache must be mapped to the socket for each of the NUMA nodes.
+# An incorrect mapping would be for the L3 cache to be assigned on per VM core.
+
+. utils.sh || {
+    echo "unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 1
+}
+
+UtilsInit
+
+if ! which lscpu; then
+    install_package util-linux
+fi
+
+lscpu --extended=cpu,node,socket,cache > lscpu.log
+# change cache column separator
+sed -i 's/:/ /g' lscpu.log
+
+# we don't need the table header
+sed -i 's/^CPU.*//g' lscpu.log
+sed -i '/^$/d' lscpu.log
+
+# table header should now look like this:
+# CPU NODE SOCKET L1d L1i L2 L3
+
+while read line;
+do
+    # Core number
+    core=$(echo "$line" | awk '{ print $2 }')
+    # NUMA Node number
+    node=$(echo "$line" | awk '{ print $2 }')
+    # Socket number
+    # socket=$(echo "$line" | awk '{ print $3 }')
+
+    l3Cache=$(echo "$line" | awk '{ print $7 }')
+
+    # Test case L3 CACHE Spread
+    if [ "$l3Cache" -ne "$node" ]; then
```

Comment:
Add test steps in comment why $2, $3, $7 are selected for the comparison. Also Failed case in the local test does not help to understand why it fails. Please add rich contents of test failure case.